vpr/src/route/route_timing.cpp

```diff
@@ -546,6 +546,24 @@ bool try_timing_driven_route_tmpl(const t_router_opts& router_opts,
 
                 routing_is_successful = true;
 
+                for (auto net_id : cluster_ctx.clb_nlist.nets()) {
+                    auto current_node = best_routing[net_id].head;
+                    std::cout << "-------\n";
+                    while (current_node != nullptr) {
+                        auto current_type = device_ctx.rr_nodes[current_node->index].type_string();
+                        auto xlow = device_ctx.rr_nodes[current_node->index].xlow();
+                        auto xhigh = device_ctx.rr_nodes[current_node->index].xhigh();
+                        auto ylow = device_ctx.rr_nodes[current_node->index].ylow();
+                        auto yhigh = device_ctx.rr_nodes[current_node->index].yhigh();
+                        if ((xlow == xhigh) && (ylow == yhigh)) {
+                            printf("Node: %d %s (%d,%d) \n", current_node->index, current_type, xlow, ylow);
```

Comment:
Not sure why we need the echo here. A few comments:
- VTR use the ``VTR_LOG()`` for messages rather than ``printf``. Please check the [vtr_log.h](https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/master/libs/libvtrutil/src/vtr_log.h) for detailed usages.
- These codes will print a message for each net that is routed. This may cause our log file very long. As a result, the regression test may take longer to run (it will read the log file line by line and extract information)
- If these codes are needed, I suggest to leave sufficient comments about why these codes are needed and how we should use it. We may output these messages to a separated log file.

Reply:
Thanks for pointing this out. I added this snippet of code for my own understanding of VTR, but it is not needed. I will remove it.